railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -34,13 +34,6 @@
 # implementation.
 # Rails.application.config.active_support.remove_deprecated_time_with_zone_name = true
 
-# Change the format of the cache entry.
-# Changing this default means that all new cache entries added to the cache
-# will have a different format that is not supported by Rails 6.1 applications.
-# Only change this value after your application is fully deployed to Rails 7.0
-# and you have no plans to rollback.
-# Rails.application.config.active_support.cache_format_version = 7.0
```

Comment:
Maybe we make it a comment that says to do it in `config/application.rb`?

Reply:
Good idea - also moved it to the bottom and added some more space around it so it stands out.